gdk/Gir.toml

```diff
@@ -240,6 +238,7 @@ status = "generate"
 [[object]]
 name = "Gdk.Device"
 status = "generate"
+final_type = false
```

Comment:
You need to move things in `src/device.rs` into a new `DeviceExtManual` trait now

Reply:
I forgot to do that.